plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -399,6 +405,18 @@ public void set(PreparedStatement statement, int index, long picosOfDay)
         };
     }
 
+    private static void writeDate(PreparedStatement statement, int index, long day)
+            throws SQLException
+    {
+        statement.setString(index, DATE_FORMATTER.format(LocalDate.ofEpochDay(day)));
+    }
+
+    private static long readDate(ResultSet results, int index)
+            throws SQLException
+    {
+        return LocalDate.parse(results.getString(index), DATE_FORMATTER).toEpochDay();
```

Comment:
I'd prefer to define this as `sqlServerDateReadFunction`, a method returning a `LongReadFunction`

Reply:
Done!

Reply:
can you please do the same for write date?

Reply:
sure, changed now